mailpoet/assets/js/src/automation/editor/components/automation/step.tsx

```diff
@@ -55,16 +55,21 @@ export function Step({ step, isSelected, context }: Props): JSX.Element {
     [step],
   );
   const { openSidebar, selectStep } = useDispatch(storeName);
+  const { context } = useContext(AutomationContext);
   const compositeState = useContext(AutomationCompositeContext);
   const { batch } = useRegistry();
 
   const compositeItemId = `step-${step.id}`;
   const stepTypeData = stepType ?? getUnknownStepType(step);
 
+  const Footer = stepType.footer;
```

Comment:
`stepType` can be undefined.

just had an automation with a trigger from another branch and opened it in your built.

Expected: To see the "Unknown trigger" panel
Current behavior: Empty screen and console error `Uncaught TypeError: stepType is undefined`

Reply:
👍 Fix: https://github.com/mailpoet/mailpoet/pull/5216/commits/f15d82a3b1a8d2ec75dbfda37d73d79775a07633